modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -5246,7 +5246,7 @@ default-preferences=Default Preferences
 default-product-display-page=Default Product Display Page
 default-regular-theme=Default Regular Theme
 default-reindex-execution-mode=Default Reindex Execution Mode
-default-reindex-execution-mode-help=This feature is under development and requires enabling feature flag LPS-177664.
+default-reindex-execution-mode-help=Select the mode that will be used by default when executing a reindex.
```

Comment:
@lipusz thoughts on changing the help text to its "final form" and not mentioning the development/feature flags any more since the options will be hidden by the OptionsProvider if FF is false?

@rbohl any word-smithing suggestions welcome if tibor approves this change

Reply:
Sounds good to me. :+1:

Reply:
> Select the mode that will be used by default when executing a reindex.

I'd just say

> Select the default reindexing mode.

If you wanted to add a sentence like "This can be overridden in the Index Actions page of the search administration panel."  that might be nice. Up to you.